bin/hipcc

```diff
@@ -684,7 +684,8 @@ if($HIP_PLATFORM eq "hcc" or $HIP_PLATFORM eq "clang"){
             $targetsStr = $ENV{HCC_AMDGPU_TARGET};
         } else {
             # Else try using rocm_agent_enumerator
-            $ROCM_AGENT_ENUM = "${ROCM_PATH}/bin/rocm_agent_enumerator";
+            $ROCMINFO_PATH = $ENV{'ROCMINFO_PATH'} // $ROCM_PATH;
+            $ROCM_AGENT_ENUM = "${ROCMINFO_PATH}/bin/rocm_agent_enumerator";
```

Comment:
@pfultz2  do you mean to change it to

```suggestion
            $ROCM_AGENT_ENUM = "rocm_agent_enumerator";
```
? Or would that break hipcc for people? Otherwise we might have to run `rocm_agent_enumerator` once to see if it works, and if not resort back to `${ROCM_PATH]/bin/rocm_agent_enumerator`?

Reply:
I am not familiar with perl, but I would think you need to use something like `$(which rocm_agent_enumerator)` but I believe the path will need to be appended as well with `$ROCM_PATH/bin` doing something like `export PATH="$PATH:$ROCM_PATH/bin`.